GTMAppAuthSwift/Sources/GTMAppAuthFetcherAuthorization.swift

```diff
@@ -18,6 +18,14 @@ import Foundation
 import AppAuthCore
 import GTMSessionFetcherCore
 
+// MARK: - Keys
+
+let authStateKey = "authState"
```

Comment:
Should we restrict the scope of these constants?  Seems like we wouldn't want them to be global.

Reply:
They aren't global; they are `internal`. Having them defined this way allows them to be used in the tests.